src/docs/get-started/flutter-for/react-native-devs.md

```diff
@@ -246,11 +246,11 @@ used to wait for a `Promise`.
 ```js
 // JavaScript
 async _getIPAddress() {
```

Comment:
Isn't it invalid JavaScript ? I would rather use `async function _getIPAddress`.

Reply:
Please consider the fact that most of the code here is designed to be inside a class, both in the case of JS and Dart. And inside a class, this is valid syntax.

Reply:
@jpaquim Thanks for these changes. Indeed this code is valid, [even with plain objects](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions).

Ideally we should refactor the examples on this page so that they run without any changes (assuming the fetch dependency is present), e.g. :

```
const example = {
  async _getIPAddress() {
    const url = "https://httpbin.org/ip";
    const response = await fetch(url);
    const responseJson = await response.json();
    const ip = responseJson.origin;
    console.log(ip);
  }
}

example._getIPAddress();
```
Any chance you can take a stab at this?